pkg/tfbridge/detailed_diff.go

```diff
@@ -497,7 +530,23 @@ func MakeDetailedDiffV2(
 	props = stripSecretsAndOutputs(props)
 	newInputs = stripSecretsAndOutputs(newInputs)
 	differ := detailedDiffer{ctx: ctx, tfs: tfs, ps: ps, newInputs: newInputs}
-	return differ.makeDetailedDiffPropertyMap(priorProps, props)
+	res := differ.makeDetailedDiffPropertyMap(priorProps, props)
+
+	if replaceOverride != nil {
+		if containsReplace(res) && !*replaceOverride {
+			for k, v := range res {
+				res[k] = demoteToNoReplace(v)
+			}
+		}
+
+		if !containsReplace(res) && *replaceOverride {
+			// We use the internal __meta property to trigger a replace when we have failed to
+			// determine the correct detailed diff for it.
+			res["__meta"] = &pulumirpc.PropertyDiff{Kind: pulumirpc.PropertyDiff_UPDATE_REPLACE}
```

Comment:
There's probably a `const` in the package for __meta, use it? https://github.com/pulumi/pulumi-terraform-bridge/blob/7917cbd3083d4d759481b80cb2b4f7ac585e251e/pkg/tfbridge/schema.go#L1026

This enables "find references".